XenAdmin/Wizards/PatchingWizard/PlanActions/RestartHostPlanAction.cs

```diff
@@ -71,7 +69,8 @@ protected override void RunWithSession(ref Session session)
                 else if (_restartAgentFallback)
                 {
                     log.Debug("Live patching succeeded. Restarting agent.");
-                    Title = string.Format(Messages.UPDATES_WIZARD_RESTARTING_AGENT, hostObj.Name());
+                    Visible = true;
+                    Title = ProgressDescription = string.Format(Messages.UPDATES_WIZARD_RESTARTING_AGENT, hostObj.Name());

```

Comment:
Do we really need to set the ProgressDescription? It seems to me like duplication because within the methods EvacuateHost, RebootHost and BringBabiesback we already set the Title and/or the Status (which is presumably why I had originally thought it would be displayed). Can we use those instead? In fact I think we might be able to lose the PlanAction's ProgressDesctription completely in favour of Status?

Reply:
Changing the Title does not trigger a change event,  so the change won't be reflected in the UI. We can fix this, but I think that the Title should ideally only be set in the constructor. Also in this case the ProgressDescription is a multi-line text containing a line for each step in this plan action, lines being added as these steps are executed. So I am not sure the Title should be changed to this. 
I haven't considered the Status property because, where it is used, it contains more detailed information than it was needed in this case and also it is only a single line describing the current operations.

Reply:
However, we have indeed a bit too many related properties in this class and we should look into combining some of them.

Reply:
Actually it seems to me the Title may not be needed at all for the plan actions (on the Patching wizard at least), it's probably there only because the plan actions were fashioned after the normal actions - but a plan action only needs to change and notify its current status and it's then up to the caller to accumulate it or do anything else with it. In any case, I agree that any simplification should be left for a different PR due to the size of the changes because they affect 2 wizards.